build/config/BUILDCONFIG.gn

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import("//build_overrides/build.gni")
```

Comment:
This introduces the contents of build.gni into the global scope. We need to avoid this because it makes it harder to integrate CHIP into other GN builds. That's why we imported these inside the private scope `(_chip_overrides`). I'd just move this import back there.

BUILDCONFIG.gn is the only file where this is a concern.

Reply:
Fixed now.